security/pfSense-pkg-suricata/files/var/db/suricata/sidmods/modifysid-sample.conf

```diff
@@ -19,5 +19,13 @@
 # "HTTP_PORTS" "HTTPS_PORTS"
 
 # multiple sids can be specified as noted below:
-# 302,429,1821 "\$EXTERNAL_NET" "\$HOME_NET"
+# 302,429,1821 "$EXTERNAL_NET" "$HOME_NET"
```

Comment:
Pretty sure these should remain escaped as it was before, and any added example should have those escaped, otherwise it's never going to match things in the rulesets.